wordpress-mobile/WordPress-Android
# 🤖 Updates Tracks Events

Fixes #19508

## Description
Updates the track events as recorder in https://github.com/wordpress-mobile/WordPress-Android/issues/19508#issuecomment-1805988116 to conform with the spec update as discussed in p1699526659899089-slack-C05NS0YV7HS

-----

## To Test:

check https://github.com/wordpress-mobile/WordPress-Android/pull/19602#issuecomment-1807407043



-----

## Regression Notes

1. Potential unintended areas of impact

N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

N/A

3. What automated tests I added (or what prevented me from doing so)

Added a new test case for a new tracking event

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
